### DISCUSSION TITLE
[Best Practices] User Management

### DISCUSSION THREAD

**Discussion by tnk-dev** [apache/kyuubi#4633] [Q&A]
Hey all,
since I am very new to Kyuubi, can you please point me to best practises how to set up user management?

Our vision as a data team:

- Data is stored as Delta in S3
- We want users to query our delta lake from tools like DBeaver with SQL
- Users should only be able to query selected/specific schemas/tables (SQL Grant?)

Questions therefore:
**where and how do we create users and passwords?**
**can you create user groups with permissions?**

**Comment by pan3793** [comment]
> where and how do we create users and passwords?

LDAP may be a good choice.

> Users should only be able to query selected/specific schemas/tables (SQL Grant?)

I hope the following link helps you.

https://kyuubi.readthedocs.io/en/master/security/authorization/spark/index.html

https://github.com/apache/kyuubi/discussions/4403

**Reply by tnk-dev** [reply]
<img width="1708" alt="image" src="https://user-images.githubusercontent.com/83520203/228596228-d424157a-2714-4e0a-889e-7e78b6833724.png">

**Reply by pan3793** [reply]
Sorry, the guide is in-progress, it should be straightforward if you have a basic idea about LDAP, please search the [configuration page](https://kyuubi.readthedocs.io/en/master/deployment/settings.html) using the keyword "LDAP".
